### PR TITLE
Klinki context roots

### PR DESCRIPTION
**Pull request by jsight**

### CODE REVIEW

**Comment by jsight** [comment]
This only fixed an issue with running FileEndpointTest, I still see a lot of failures, though.

**Comment by jsight** [comment]
Ok, I think all of the tests are fixed now. :)

File: tests/test-authentication-client/src/main/resources/keycloak.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "realm": "rhamt",
   "auth-server-url": "http://localhost:8180/auth",
-  "resource": "rhamt-web/api",
+  "resource": "rhamt-web",
```
Comment (klinki) [inline tests/test-authentication-client/src/main/resources/keycloak.json]: ah, I missed this one. Even though I fixed it almost everywhere else... Thanks :)

**Comment by klinki** [comment]
@jsight Thanks Jess! I had to rebase, but I cherry-picked your changes :)

**Pull request closed without merging**